js/components/mixing-view.js

```diff
@@ -140,7 +141,8 @@ export default class MixingView extends PureComponent {
             aPegged={aTemp}
             bPegged={bTemp}
             forceCover={true}
-          />
+            mixingValues={{ aTemp, bTemp }}
```

Comment:
I think you should be able to use `aPegged` and `bPegged` here instead of adding a new param...I probably could have named those values more clearly

Reply:
Fixed, thanks!